cfgov/jinja2/v1/youth_employment_success/index.html

```diff
@@ -34,7 +34,10 @@
 {% block javascript scoped %}
     {{ super() }}
     <script async>
-      if ( document.body.parentElement.className.indexOf( 'no-js' ) === -1 ) {
+      var htmlEl = document.documentElement;
+
+      if (htmlEl.className.indexOf('no-js') < 0) {
```

Comment:
It looks like the feature detection check has been removed, so I'm still not clear why this is needed at all? You should be able to target styles via the no-js on the `html` element with, e.g.:

```
.no-js .my-custom-class { /* thing to do when in the no-js state */ }

.js .my-custom-class { /* thing to do when in the js state */ }
```

Reply:
Don't we not want to load the custom JS for this tool in the case of an unsupported environment?

Reply:
Oh wait, I see `document.documentElement` IS the `html` element, so this should be good. I was thinking you were still targeting `main`. This is probably a good change to do overall. I was thinking we targeted the parentElement of body for compatibility reasons, but it doesn't look like we need to https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement#Browser_compatibility

Why check if the index is less than zero versus checking if it's -1?

Reply:
I guess slightly less characters? Originally I didn't have the `htmlEl` as a separate variable, everything was inline, and I think the resultant line was longer than 80 characters 😄